lambda/create/set_defaults.go

```diff
@@ -3,7 +3,7 @@ package main
 import "github.com/ministryofjustice/opg-data-lpa-store/internal/shared"
 
 func SetDefaults(input shared.LpaInit) shared.LpaInit {
-	activeAttorneyCount, replacementAttorneyCount := countAttorneys(input.Attorneys, input.TrustCorporations)
+	activeAttorneyCount, replacementAttorneyCount := shared.CountAttorneys(input.Attorneys, input.TrustCorporations)
```

Comment:
is it worth changing activeAttorneyCount to originalAttorneyCount as an originalAttorney can be made inactive.